Microsoft.HealthVault/Thing/ThingBase.cs

```diff
@@ -1237,7 +1237,7 @@ private void AddXmlSectionValues(XPathNavigator thingNavigator)
             // Check for the "data-xml" data-group
 
             XPathNodeIterator dataXmlIterator =
-                thingNavigator.Select("data-xml");
+                thingNavigator.Select("thing/data-xml");
```

Comment:
Why was it necessary to add "thing/"?  Wouldn't it have broken everything if it wasn't working?  This **is** on the ThingBase, after all.  Nothing uses "thing/" to look up their xml, it looks like they all just start with an XPathNavigator("thing") and work from there.